ui/src/app/shared/components/urls.ts

```diff
@@ -30,7 +30,7 @@ export function revisionUrl(url: string, revision: string): string {
     let urlSubPath = isSHA(revision) ? 'commit' : 'tree';
 
     if (url.indexOf('bitbucket') >= 0) {
-        urlSubPath = isSHA(revision) ? 'commits' : 'branch';
+        urlSubPath = isSHA(revision) ? 'commits' : 'src';
```

Comment:
Good find! Would you mind opening another PR with just this change? I'd like to cherry-pick the fix back to previous versions.

Reply:
NP

Reply:
Oh, and have you confirmed that this still works for URLs _without_ paths? e.g. https://bitbucket.org/Nirshtain2065/yamls/HEAD

Reply:
https://github.com/argoproj/argo-cd/pull/10862

Reply:
@crenshaw-dev I verified it and add new video in the new PR while I am clicking `HEAD`